app/models/miq_group.rb

```diff
@@ -17,7 +16,7 @@ class MiqGroup < ActiveRecord::Base
   virtual_column :read_only,          :type => :boolean
   virtual_column :user_count,         :type => :integer
 
-  # TODO: Commented out :ui_task_set_idfor work for implementing new RBAC design. This should be replaced with miq_group_id.
+  # TODO: Commented out :ui_task_set_id for work for implementing new RBAC design. This should be replaced with miq_group_id.
   validates_presence_of   :description, :guid #, :ui_task_set_id
```

Comment:
I think the comment may be wrong about `miq_group_id`.  Roles based on UiTaskSet were replaced by miq_user_role so it seems we would validate `miq_user_role_id`.  Maybe @gtanzillo can verify.